modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js

```diff
@@ -73,45 +75,53 @@ const LocalizableText = ({
 		: predefinedValue;
 
 	useEffect(() => {
-		const translationManager = Liferay.component('translationManager');
+		let locales = currentAvailableLocales;
```

Comment:
Variable `locales` is only used inside the `if (!localizedObjectField)` block, only to be reassigned to `availableLocales`, so it is doing nothing. When doing `setCurrentAvailableLocales`, pass `availableLocales` directly.

Reply:
Done!